app/assets/javascripts/app/utils/pluralize.js

```diff
@@ -0,0 +1,6 @@
+export default (word, count) => {
+  const I18n = window.LoginGov.I18n;
+  const tString = I18n[word] || word;
+
+  return `${tString}${count !== 1 ? 's' : ''}`;
```

Comment:
😬 this is not i18n-safe either (and won't work on irregular plurals like baby ➡️ babies right?)

The most common way to store multiple keys, often nested. Here's how Rails does this: http://guides.rubyonrails.org/i18n.html#pluralization

So we'd have:

```
time:
  minute:
    one: 'minute'
    other: 'minutes'
```

Other languages have more complex rules than "special case for one" so this allows for that structure.